cluster/sync.sh

```diff
@@ -19,25 +19,41 @@ set -ex
 source ./cluster/kubevirtci.sh
 kubevirtci::install
 
-registry_port=$(./cluster/cli.sh ports registry | tr -d '\r')
-registry=localhost:$registry_port
+if [[ "$KUBEVIRT_PROVIDER" == external ]]; then
+    if [[ ! -v DEV_REGISTRY ]]; then
```

Comment:
I think `$` is missing before `DEV_REGISTRY`

Reply:
Nope, since it's checking the existence of a var name